### PR TITLE
AV-134285 : Documentation for L4Rule CRD

### PR DESCRIPTION
**Pull request by swathinsankaran**
This PR contains the document changes for L4Rule CRD.

### CODE REVIEW

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by swathinsankaran** [comment]
build ako

**Review by akshayhavile — COMMENTED** [review]
@swathinsankaran: Should we mention that it is not supported for Gw objects in L4 context.

File: docs/crds/l4rule.md [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+        throttle: 30
+```
+
+The field `throttle` supports values from **0** to **65535** and will be in effect only when `enabled` is set to **true**. AKO defaults the value of `throttle` to **10**. Set the value of `throttle` to **Zero (0)** to deactivate throttling.
+
+By default, the AKO sets the `duration` of logging the non-significant logs to **30 minutes**. The user has to configure the `duration` as **Zero (0)** to capture the logs indefinitely.
+
+#### Express custom Application Profiles
+
+L4Rule CRD can be used to express application profile references. The application profile must be created in the AVI Controller before referring to it. The application profile should be of `TYPE` of `APPLICATION_PROFILE_TYPE_L4`.
```
[the remark this answers is not in the document]
Comment (swathinsankaran) [reply docs/crds/l4rule.md]: I have addressed the review comment.

File: docs/crds/l4rule.md [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+
+```yaml
+    applicationProfile: Custom-L4-Application-Profile
+ ```
+
+Currently, the application profile of type `APPLICATION_PROFILE_TYPE_L4` is supported. The application profile can be used to enable PROXY Protocol, rate limit the connections from a client, etc.
```
[the remark this answers is not in the document]
Comment (swathinsankaran) [reply docs/crds/l4rule.md]: I have removed the duplicate sentence.

File: docs/crds/l4rule.md [hunk 3]
```diff
@@ -0,0 +1,379 @@
...
+
+```yaml
+    securityPolicyRef: Custom-Security-Policy
+```
+
+The Security Policy must be created in the AVI Controller before referring to it.
+
+#### Express custom datascripts
+
+The L4Rule CRD can be used to express datascript references. The datascript references should have been created in the AVI Controller before the CRD creation.
```
[the remark this answers is not in the document]
Comment (swathinsankaran) [reply docs/crds/l4rule.md:182]: yes, we support list of datascript references.

**Comment by swathinsankaran** [comment]
@akshayhavile, I have added a note that we are not supporting L4Rule CRD with Gateway.

**Comment by swathinsankaran** [comment]
build ako

**Review by arihantg — APPROVED** [review]

**Pull request merged**